CMakeLists.txt

```diff
@@ -282,3 +282,24 @@ endif()
 set(docfiles COPYING NEWS AUTHORS)
 install(FILES ${docfiles}
         DESTINATION "${DOCDIR}")
+
+################################################################################
+# pkg-config support
+################################################################################
+if(UNIX)
+  set(SQLITE3_LIBS -lsqlite3)
+  if(TIFF_ENABLED)
+    set(TIFF_LIBS -ltiff)
+  endif()
+  if(CURL_ENABLED)
+    set(CURL_LIBS -lcurl)
+  endif()
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/proj.pc.cmake.in
+    ${CMAKE_CURRENT_BINARY_DIR}/proj.pc
+    @ONLY)
+
+  install(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/proj.pc
+    DESTINATION lib/pkgconfig)
```

Comment:
I guess Debian will be unhappy about this rule. Due to multi-arch, `proj.pc` resides in `/usr/lib/x86_64-linux-gnu/pkgconfig/`. In a `GNUInstallDirs` context, the destination would probably be `${CMAKE_INSTALL_LIBDIR}/pkgconfig`.